internal/gatewayapi/translator.go

```diff
@@ -129,6 +133,15 @@ func (t *Translator) Translate(resources *Resources) *TranslateResult {
 	// Get Gateways belonging to our GatewayClass.
 	gateways := t.GetRelevantGateways(resources.Gateways)
 
+	// Get Routes belonging to our GatewayClass.
+	routes := t.GetRelevantRoutes(gateways,
```

Comment:
I dont think this is needed, L160-L170 already compute the relevant routes today e.g.
https://github.com/envoyproxy/gateway/blob/2ecc54da663dc10c9257a61843cd14ba1bb77dcc/internal/gatewayapi/route.go#L63
we just need to append it into a list and pass it over